Lib/fontbakery/profiles/post.py

```diff
@@ -275,5 +280,13 @@ def x_leftmost_intersection(paths, y):
                 ("Font is not italic, so post.italicAngle should be equal to zero."),
             )
 
+    if bad_glyphs:
+        yield WARN, Message(
+            "empty-glyphs",
+            ("The following glyphs were present but did not contain any outlines: "+
```

Comment:
Do you have a sample font file for this WARN?

Reply:
I don't, but we can make one from a test font by doing:

```
font["glyf"]["I"].endPtsOfContours = []
font["glyf"]["I"].coordinates = []
font["glyf"]["I"].flags = []
font["glyf"]["I"].numberOfContours = 0
```